conda/recipes/legate-boost/meta.yaml

```diff
@@ -80,12 +79,12 @@ requirements:
     {% if gpu_enabled_bool %}
     - cuda-version ={{ cuda_version }}
     - cuda-cudart-dev
-    - cunumeric {{ legate_version }} =*_gpu
-    - legate {{ legate_version }} =*_gpu
+    - cunumeric {{ legate_version }} =*_gpu*
+    - legate {{ legate_version }} =*_gpu*
```

Comment:
`*_gpu` = "ends with `_gpu`"

`*_gpu*` = "contains `_gpu` somewhere"

I think we want that slightly more flexible form, as there's no guarantee that the packages we install will always end with `_gpu` (even if the packages we currently *build* do).

Look at the diff from the PR that added these build string components to `legate` packages... there are some other things at the end of the build strings sometimes: https://github.com/nv-legate/legate.core.internal/pull/1164/files